src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -57,7 +60,7 @@ public TaskCard(Task task, int displayedIndex) {
         deadline.setText("Deadline: " + task.getDeadline().toString());
         email.setText("Email: " + task.getPersonEmailAddress().toString());
```

Comment:
I think dont need to include email already since u included person name

Reply:
Email label removed from Task Card